src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -1,9 +1,48 @@
+<!-- 
```

Comment:
Please add `{#`  here, to indicate for jinja2 it should not replace here something.  Otherwise I get the error

```
  File "/home/<user>/programs/anaconda3/lib/python3.9/site-packages/cryptoadvance/specter/templates/includes/tx-table.html", line 35, in top-level template code
    wallet="{{ wallet.alias }}"
  File "/home/<user>/programs/anaconda3/lib/python3.9/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'wallet' is undefined
```